src/components/Block.vue

```diff
@@ -437,12 +437,22 @@ function parseMarkdown (event:KeyboardEvent) {
     props.block.details.value = newContent
   }
 
+  const handleQuoteContent = () => {
+    emit('setBlockType', BlockType.Quote)
+    const newContent = textContent.replace(/^>\s(.*)$/, '$1')
+    ;(content.value as any).innerText = newContent
+    props.block.details.value = newContent
+  }
+
+
```

Comment:
Thanks for this!

Can we combine this into `headingRegexpMap` and `handleHeadingContent` just above it? Then we can rename the following
- `handleHeadingContent` -> `handleMarkdownContent`
- `headingRegexpMap` -> `markdownRegexpMap`

Reply:
Of course!